### PR TITLE
refactor: refactor test cases for Params

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
### Params: TestQueryParams
Add a test case because we have not tested for Params.

```
// QueryParamsResponse is the response type for the Query/Params RPC method.
message QueryParamsResponse {
  // params defines the parameters of the module.
  Params params = 1 [ (gogoproto.nullable) = false ];
}
```

spec:
https://github.com/Finschia/wasmd/blob/main/proto/cosmwasm/wasm/v1/query.proto#L63

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly. (not needed)
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` (not needed)

### CODE REVIEW

File: x/wasm/keeper/querier_test.go [hunk 1]
```diff
@@ -798,31 +798,51 @@ func TestQueryPinnedCodes(t *testing.T) {
 }
 
 func TestQueryParams(t *testing.T) {
-	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
-	keeper := keepers.WasmKeeper
-
-	q := Querier(keeper)
+	specs := map[string]struct {
+		srcPermission types.AccessType
+		expInstConf   types.AccessConfig
```
Comment (170210) [inline x/wasm/keeper/querier_test.go:803]: I think these two should be renamed, because they are both src and expected.

**Review by 170210 — COMMENTED** [review]
I think the description needs to be changed, because this pr just adds tests for other permissions.

File: x/wasm/keeper/querier_test.go [hunk 2]
```diff
@@ -798,31 +798,51 @@ func TestQueryPinnedCodes(t *testing.T) {
 }
 
 func TestQueryParams(t *testing.T) {
-	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
-	keeper := keepers.WasmKeeper
-
-	q := Querier(keeper)
```
Comment (170210) [inline x/wasm/keeper/querier_test.go:804]: I think it is better to setting the dummy environment here than in loop.

Comment (kokeshiM0chi) [reply x/wasm/keeper/querier_test.go]: I fixed it.

File: x/wasm/keeper/querier_test.go (hunk 1 above)
Comment (kokeshiM0chi) [reply x/wasm/keeper/querier_test.go]: Thanks. I fixed it.

File: x/wasm/keeper/querier_test.go [hunk 3]
```diff
@@ -871,17 +871,39 @@ func TestQueryParams(t *testing.T) {
 	require.Equal(t, paramsResponse.Params.CodeUploadAccess, defaultParams.CodeUploadAccess)
 	require.Equal(t, paramsResponse.Params.InstantiateDefaultPermission, defaultParams.InstantiateDefaultPermission)
```
Comment (170210) [inline x/wasm/keeper/querier_test.go:872]: I think you can delete from L865-L872, because it is the test about DefaultParams.

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +14,7 @@
 * [\#73](https://github.com/Finschia/wasmd/pull/73) test: add the check for expPaginationTotal
 * [\#72](https://github.com/Finschia/wasmd/pull/72) add pagination next key test in ContractHistory
 * [\#75](https://github.com/Finschia/wasmd/pull/75) test: add the test case for InactiveContract
+* [\#69](https://github.com/Finschia/wasmd/pull/69) refactor: add test cases to Params
```
Comment (da1suk8) [inline CHANGELOG.md:17]: Do we need a prefix here?
(Some of them are already merged...)

File: x/wasm/keeper/querier_test.go (hunk 3 above)
Comment (kokeshiM0chi) [reply x/wasm/keeper/querier_test.go]: I fixed it.

File: CHANGELOG.md (hunk 4 above)
Comment (kokeshiM0chi) [reply CHANGELOG.md:17]: Thanks. The change.log will be refactored together at the end. Then remove the prefix.

**Review by da1suk8 — DISMISSED** [review]

**Review by 170210 — APPROVED** [review]

**Review by da1suk8 — APPROVED** [review]

**Pull request merged**